### PR TITLE
[HAMMER] [Changelog] Update for Hammer Beta-1 release

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
@miq-bot add_label documentation

**Comment by JPrause** [comment]
@miq-bot assign simaishi

**Comment by JPrause** [comment]
@carbonin can you review and Satoe will merge.

**Comment by simaishi** [comment]
@carbonin Doesn't this contain some PRs only in `master` branch (like Container orchestrator)?

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,34 @@ All notable changes to this project will be documented in this file.
...
+### Added
+- Backport enhancements made to master since Gaprindashvili [(#291)](https://github.com/ManageIQ/manageiq-pods/pull/291)
+- Adding support for OpenID-Connect [(#251)](https://github.com/ManageIQ/manageiq-pods/pull/251)
+- Use the new repmgr preload library name [(#300)](https://github.com/ManageIQ/manageiq-pods/pull/300)
+- Use the ruby 2.4 image [(#305)](https://github.com/ManageIQ/manageiq-pods/pull/305)
+- Updates for Hammer branch release [(#306)](https://github.com/ManageIQ/manageiq-pods/pull/306)
+- Add a simple script to deploy MIQ on minishift [(#303)](https://github.com/ManageIQ/manageiq-pods/pull/303)
+- Remove 15 second sleep from startup time [(#270)](https://github.com/ManageIQ/manageiq-pods/pull/270)
+- Add artemis service [(#268)](https://github.com/ManageIQ/manageiq-pods/pull/268)
+- Container orchestrator [(#207)](https://github.com/ManageIQ/manageiq-pods/pull/207)
```
Comment (carbonin) [inline CHANGELOG.md]: Yeah, I don't think this was included in Hammer

**Comment by carbonin** [comment]
Good catch @simaishi

**Comment by simaishi** [comment]
@JPrause I think we created `hammer` branch off of `gaprindashvili` branch for this repo, so it's probably easier if you take a look at commits in `hammer` and generate the changelog. We don't have that many commits.

**Comment by JPrause** [comment]
@simaishi will do

**Pull request closed without merging**